script.js

```diff
@@ -35,4 +35,283 @@ libutton3.addEventListener('click', () => {
+SeeButton2.setAttribute('data-target', '#'.concat(projects[1].id));
+project2Desk.appendChild(SeeButton2);
+
+const project2DeskImage = document.querySelector('#projects-desktop div.div10 img');
+project2DeskImage.src = projects[1].featImage;
+
+// Generate PopUp Desktop Windows
+const popUpDeskList = document.querySelector('#popwindesktop');
+const popUpDeskTemplate = popUpDeskList.querySelector('div.interndesk');
+
+for (let i = 0; i < 2; i += 1) {
+  const clonedPopUpDesk = popUpDeskTemplate.cloneNode(true);
+  clonedPopUpDesk.id = projects[i].id;
+  const h2PopDesk = clonedPopUpDesk.querySelector('.poph2desk');
+  h2PopDesk.textContent = projects[i].name;
+
+  const project3Buttons = clonedPopUpDesk.querySelector('.b3popdesk');
+  for (let j = 0; j < projects[i].technologies.length; j += 1) {
+    const Button = document.createElement('button');
+    Button.className = 'sbuttonpopdesk';
+    Button.textContent = projects[i].technologies[j];
+    project3Buttons.appendChild(Button);
+  }
+
+  const projectImage = clonedPopUpDesk.querySelector('.popmaindesk');
+  projectImage.src = projects[i].featImage;
+  const pPopWinDesk = clonedPopUpDesk.querySelector('.popp1desk');
+  pPopWinDesk.textContent = projects[i].description;
+
+  const b1PopWinDesk = clonedPopUpDesk.querySelector('.popb1desk');
+  b1PopWinDesk.setAttribute('onclick', `window.location.href='${projects[i].liveLink}';`);
+  const b2PopWinDesk = clonedPopUpDesk.querySelector('.popb2desk');
+  b2PopWinDesk.setAttribute('onclick', `window.location.href='${projects[i].sourceLink}';`);
+
+  popUpDeskList.appendChild(clonedPopUpDesk);
+}
+
+const popupdesk1 = document.querySelector('#popwindesktop #project1');
+const popupdesk2 = document.querySelector('#popwindesktop #project2');
+const hiddendeskpop = document.querySelector('body div.hide_desktop');
+
+SeeButton.addEventListener('click', () => {
+  popupdesk1.classList.replace('hide_desktop', 'show_desktop');
+  MobileSec1.classList.replace('show_mobile', 'hide_mobile');
+  hiddendeskpop.classList.replace('hide_desktop', 'show_desktop');
+});
+
+SeeButton2.addEventListener('click', () => {
+  popupdesk2.classList.replace('hide_desktop', 'show_desktop');
+  MobileSec1.classList.replace('show_mobile', 'hide_mobile');
+  hiddendeskpop.classList.replace('hide_desktop', 'show_desktop');
+});
+
+// create event for close 'x' buttons of desktop popups
+const closedeskbutton1 = popupdesk1.querySelector('.popxdesk');
+const closedeskbutton2 = popupdesk2.querySelector('.popxdesk');
+
+const listclosedeskbuttons = [closedeskbutton1, closedeskbutton2];
+const listdeskpopups = [popupdesk1, popupdesk2];
+
+for (let i = 0; i < listclosedeskbuttons.length; i += 1) {
+  listclosedeskbuttons[i].addEventListener('click', () => {
+    listdeskpopups[i].classList.replace('show_desktop', 'hide_desktop');
+    MobileSec1.classList.replace('hide_mobile', 'show_mobile');
+    hiddendeskpop.classList.replace('show_desktop', 'hide_desktop');
+  });
```

Comment:
- [ ] You are generating your popup window 😊 That's great 👍 But the thing is it would be better if you would generate your popup window dynamically because you can see that you have only 5 cards and see your code how big your code is. Let's suppose you have 100 cards then how much big your code would be 😄 So kindly generate your popup dynamically 👍

Hint: You can see this [pull request](https://github.com/HAMMAS-SALEEM/Portfolio/pull/16/files#diff-e07d531ac040ce3f40e0ce632ac2a059d7cd60f20e61f78268ac3be015b3b28f) to get an idea what i am talking about 👍